Added command to run python script to follow first script example

sadik312 opened the pull request:
### **User description**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to review and merge it quickly**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, and help reviewers by making them as simple and short as possible.


<!--- Provide a general summary of your changes in the Title above -->

### Description
<!--- Describe your changes in detail -->
I have added the command to run the selenium file using Python.

### Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
This was missing from the documentation.
### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Change to the site (I have double-checked the Netlify deployment, and my changes look good)
- [ ] Code example added (and I also added the example to all translated languages)
- [ x] Improved translation
- [x] Added new translation (and I also added a notice to each document missing translation)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
- [ x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [x] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.
<!--- Provide a general summary of your changes in the Title above -->


___

### **PR Type**
documentation, enhancement


___

### **Description**
- Added a section in `examples/python/README.md` to provide users with the command to run a Python script, enhancing the usability of the examples.
- Updated `website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md` to link directly to the newly added instructions in the examples README, improving the documentation's interactivity and user guidance.



___



### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>README.md</strong><dd><code>Add Python Script Execution Instructions</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
examples/python/README.md

<li>Added instructions on how to execute a Python script using the command <br>line.<br>


</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1710/files#diff-06893968f9273904f8e5ac010d61b3b4dacb57273629c03f2c6592db56c6f506">+8/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>first_script.en.md</strong><dd><code>Link Python Script Execution in Documentation</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>
      
website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md

<li>Updated the Python tab to link to the new code block in the examples <br>README.<br>


</details>
    

  </td>
  <td><a href="https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1710/files#diff-0757312e3d52b8201ca35b35c4bdad191db3e0978681a45c0cad1bebeb49fd3c">+1/-1</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

sadik312 committed:
added command to run python script to follow first script example

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/SeleniumHQ/seleniumhq.github.io?pullRequest=1710) <br/>All committers have signed the CLA.

harsha509 reviewed (changes requested):
HI @sadik312 ,

Thank you the PR. Could you please make similar changes to other translated files too.

Thanks,
Sri

sadik312 commented:
> HI @sadik312 ,
> 
> Thank you the PR. Could you please make similar changes to other translated files too.
> 
> Thanks, Sri

Hi @harsha509 , 
Yes, I will make the changes to the other files too.
Thanks

sadik312 committed:
added execution commands for all languages to follow script example

sadik312 reviewed (commented):
I have added README.md files for the examples for: 
- dotnet
- java
- kotlin

I have added the file execution command for:
- python

I have also reflected all of the changes in the `first_script.en.md` file

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

harsha509 reviewed (changes requested):
Hi @sadik312 ,

Thank you for the changes. However, there seem to be some conflicts within the PR. Please could you resolve these when you get a chance?

Thanks,
Sri

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 commented:
Hi @harsha509,
I have fixed the conflict. I added the guide for Gradle and someone else added the guide for Maven and I have merged the 2 together. Please let me know if there are any other issues.
Thank you,
Sadik

harsha509 commented on examples/dotnet/README.md:
````diff
@@ -0,0 +1,36 @@
+3. Install dependecies using dotnet
+
+```
+dotnet restore
````
So, this doc doesn't say/explain on installing and using the `dotnet` command. Adding prerequisites on dotnet installation steps could be beneficial. Any thoughts ?

This comment applies for all other lang readme.md!

sadik312 replied:
That does seem like a good idea, however, would the prerequisite also need to include steps for installing on different OS's? For instance, one set of steps for MacOS, one set of steps for Windows.

harsha509 replied:
HI @sadik312 ,

Pointing to [this link on Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/core/install/) should be sufficient, as it covers all cross-platform installation steps for .NET

sadik312 replied:
Hi @harsha509,
I have added the Microsoft Learn link and solved the conflict.
Thanks

harsha509 commented:
Hi @sadik312 ,

Also could you please resolve conflicts!

Thanks,
Sri

sadik312 committed:
added microsoft .NET cross-platform installation link

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

A1exKH reviewed (approved):
@sadik312 LGTM.

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 commented:
Hi @harsha509,
Would you be able to approve this PR?
Thank you

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

A1exKH reviewed (approved):
LGTM.

sadik312 commented:
Hi @harsha509,
Just wanted to check if you could review my PR when you have some time. Let me know if any changes are needed. Thank you

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

sadik312 committed:
Merge branch 'trunk' into sadik-feature-branch

diemol committed:
Resolve conflicts with trunk

diemol committed:
Merge branch 'trunk' into sadik-feature-branch

diemol committed:
Fix inaccurate content from Qodo review and wire up docs

- dotnet: move the single-script section before the SeleniumDocs cd step,
  since HelloSelenium.cs lives one directory up (Qodo: wrong path)
- java: drop the added Gradle/javac section - this project uses Maven, and
  the referenced FirstScript.js file doesn't exist (Qodo: wrong build tool
  and filename); trunk's existing Maven instructions already cover this
- kotlin: replace the kotlinc/java -jar section with the actual working
  command, since FirstScriptTest.kt is a JUnit test class, not a
  standalone script (Qodo: wrong path, but also not runnable that way)
- first_script.en.md: wire up the CSharp and Kotlin "Running Selenium
  File" tabs to the corrected content, and fix the CSharp line reference
  that had drifted after the dotnet README was restructured

diemol committed:
Address remaining Qodo findings

- dotnet: clarify that HelloSelenium.cs needs the .NET 10 SDK, since the
  general prerequisite only listed 8.0+ for the test suite
- kotlin: rename "Execute a kotlin script" to "Run a single Kotlin example"
  since the command runs a JUnit test via Maven, not a standalone script
- first_script.en.md: update the CSharp line reference after the dotnet
  README prerequisites section grew by two lines

diemol committed:
Fix Kotlin README heading hierarchy per Qodo review

Demote "Run a single Kotlin example" from an H1 to an H2 so it reads as
a subsection rather than a separate document, and tighten the wording.
